pcweb/pages/docs/advanced_guide/api_routes.py

```diff
@@ -43,4 +43,33 @@ def api_routes():
         doctext(
             "This is useful for creating a backend API that can be used for purposes other than your Pynecone app. ",
         ),
+        subheader("Reserved Routes"),
+        doctext(
+            "Some routes on the backend are reserved for the runtime of Pynecone, and should not be overriden unless you know what you are doing"
+        ),
+        subheader("Ping", level=2),
+        doctext(
+            pc.code("localhost:8000/ping/"),
+            ": You can use this route to check the health of the backend.",
+        ),
+        doctext(
+            "The expected return is ",
+            pc.code('"pong"'),
+        ),
+        subheader("Event", level=2),
+        doctext(
+            pc.code("localhost:8000/event"),
+            ": the frontend will use this route to notify the backend that an event occurred",
+        ),
+        pc.alert(
```

Comment:
we can use the `docalert` component for consistent styling (can do in a follow up)